packages/react-dom/src/__tests__/ReactDOMServerLifecycles-test.js

```diff
@@ -225,8 +225,10 @@ describe('ReactDOMServerLifecycles', () => {
       }
     }
 
-    expect(() => ReactDOMServer.renderToString(<Component />)).toWarnDev(
-      'componentWillMount has been renamed',
+    expect(() => ReactDOMServer.renderToString(<MyComponent />)).toWarnDev(
+      'componentWillMount has been renamed, and is not recommended for use. See https://fb.me/react-unsafe-component-lifecycles for details.\n\n' +
+        '* Move code from componentWillMount to componentDidMount (preferred in most cases) or the constructor.\n\n' +
+        'Please update the following components: MyComponent',
```

Comment:
I'm not sure I understand why we updated this test. This test isn't about verifying warnings. It's testing behavior, and the `toWarnDev()` assertions are just there to suppress the (expected) warnings.

Reply:
Hm, let me check the git blame for when this warning was added. Might be I got confused or the warning message wasn't checked in the first place.

Reply:
No worries. I think maybe just revert this file and I'll merge the rest?

Reply:
Ok so the reason I chose this test was that it was the first one I found that hit https://github.com/facebook/react/blob/4b5cb6bddfc7b25ad41f4c5a555b40250479edb7/packages/react-dom/src/server/ReactPartialRenderer.js#L577. But I agree that this is the wrong place. There's already a better place. 0ed6d5d90962c983e7ce07ff8fb773110c19afb2 should resolve this issue.

Reply:
Agreed, the new location seems better.